### ISSUE TITLE
brio update from CRAN

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/brio#28]
```
Version: 1.1.3
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    read_file.c:29:32: warning: format ‘%i’ expects argument of type ‘int’, but argument 2 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
    read_file_raw.c:29:32: warning: format ‘%i’ expects argument of type ‘int’, but argument 2 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
    read_lines.c:67:32: warning: format ‘%i’ expects argument of type ‘int’, but argument 2 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
  See ‘/home/hornik/tmp/R.check/r-devel-gcc/Work/PKGS/brio.Rcheck/00install.out’ for details.
  * used C compiler: ‘gcc-13 (Debian 13.2.0-5) 13.2.0’
Flavor: [r-devel-linux-x86_64-debian-gcc](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/brio-00check.html)
```

**Comment by gaborcsardi** [comment]
Issue fixed in 97be018b08c9b5d447f20b6cdb3ddaa48f245ba9.

**Comment by gaborcsardi** [comment]
Also, on Windows:
```
    brio.c:26:49: warning: conversion lacks type at end of format [-Wformat=]
    brio.c:26:11: warning: too many arguments for format [-Wformat-extra-args]
```
